### PR TITLE
fix

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: conf/.env [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#
+DATABASE_URL=mysql://__DB_USER__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__?charset=utf8mb4&serverVersion=mariadb-10.5.19
+
+#================================================================================
+# The full documentation can be found at https://www.kimai.org/documentation/emails.html
+#
+# Email will be sent with this address as sender:
+MAILER_FROM=admin@__DOMAIN__
+# Email connection (disabled by default) - see documentation for the format
+MAILER_URL=smtp://localhost:25?encryption=&auth_mode=
```
Comment (alexAubin) [inline conf/.env:30]: Note for the future :  we may want mails to be dkim-signed and may use `MAILER_URL="smtp://<user>:<password>@localhost:25?encryption=&auth_mode="` syntax (cf their doc) but that requires packaging v2 with `allow_email=true` on the system user resource

File: scripts/install [hunk 2]
```diff
@@ -105,7 +105,7 @@ setfacl -R -m g:"www-data":rwX -m u:$app:rwX "$final_path/var/"
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=1
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
+ynh_add_fpm_config --usage=low --footprint=low
```
Comment (alexAubin) [inline scripts/install:108]:
```suggestion
ynh_add_fpm_config
```

cf recent changes in ynh_add_fpm_config, we don't need to specify `--usage=low --footprint=low` as long as the extra_php-fpm.conf file exists

File: conf/.env (hunk 1 above)
Comment (alexAubin) [inline conf/.env:30]: Note for the future :  we may want mails to be dkim-signed and may use `MAILER_URL="smtp://<user>:<password>@localhost:25?encryption=&auth_mode="` syntax (cf their doc) but that requires packaging v2 with `allow_email=true` on the system user resource

File: scripts/upgrade [hunk 3]
```diff
@@ -135,7 +135,7 @@ ynh_install_app_dependencies $pkg_dependencies
 ynh_script_progression --message="Upgrading PHP-FPM configuration..." --weight=1
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
+ynh_add_fpm_config --usage=low --footprint=low
```
Comment (alexAubin) [inline scripts/upgrade:138]:
```suggestion
ynh_add_fpm_config
```

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**